cpp/CMakeLists.txt

```diff
@@ -18,8 +18,13 @@ option(OPT_MSGPACK "Serialization using msgpack " OFF)
 # Bindings
 option(PYTHON_BINDINGS "Build bindings for Python" ON)
 set(PYTHON_VERSION 3.7 CACHE STRING "Build bindings for Python version")
+set(BUILD_WITH_PINOCCHIO_PYTHON false CACHE BOOL "Force compile with Pinocchio bindings")
+# This option is needed because sometime the flag
+# PINOCCHIO_WITH_PYTHON_INTERFACE is not properly set depsite a python
+# binding is installed.
```

Comment:
This comes from robotpkg installation and is a bug from Pinocchio. I will report it.